src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -482,6 +482,16 @@
        </attribute>
       </widget>
      </item>
+    <item row="9" column="3">
```

Comment:
The introduction of a 3rd column throws off the positioning/aligning of the existing options. Instead, like with the tooltips options, use a horizontal layout for the time display options.